### DISCUSSION TITLE
AAC low volume

### DISCUSSION THREAD

**Discussion by ArturNadolski** [MusicPlayerDaemon/MPD#1163] [General]
Example stream: https://stream.open.fm/11

Stream info from VLC:
AAC(mp4a)
sample rate: 44100 Hz
bits per sample: 32
AAC extension: SBR+PS

AAC streams are at very low volume corespondig to other streams (mp3, flac).
Any suggestion how to "normalize" volume switching between mp3 stream and aac? Maybe bump up volume olny for AAC streams?

**Comment by MaxKellermann** [comment]
There's the "normalize" filter plugin: https://mpd.readthedocs.io/en/stable/plugins.html#normalize
.. which can be enabled with the global option `volume_normalization`

**Reply by ArturNadolski** [reply]
But is it realy normalization between switched streams with different codecs?
Try to switch from: https://play.sas-media.ru/play_256.m3u to : https://stream.open.fm/11 and notice volume level change.
